packages/eslint-plugin-svelte/src/utils/svelte-context.ts

```diff
@@ -0,0 +1,201 @@
+import type { RuleContext } from '../types.js';
+import fs from 'fs';
+import path from 'path';
+import { getPackageJson } from './get-package-json.js';
+import { getFilename, getSourceCode } from './compat.js';
+
+const isRunInBrowser = !fs.readFileSync;
+
+export type SvelteContext = {
+	svelteVersion: '3/4' | 5;
+	fileType: '.svelte' | '.svelte.[js|ts]';
+	runes: boolean;
+	svelteKitVersion: '1-next' | 1 | 2 | null;
```

Comment:
dtto about string and number literals.

Reply:
updated! https://github.com/sveltejs/eslint-plugin-svelte/commit/531911a246216a0fcbb6993ccb4a12576b01eb7c